### PR TITLE
Comments for opening hours

### LINKED ISSUE #236: Als Gemeente wil ik een opmerking opnemen bij de openingstijden
Als Gemeente wil ik een opmerking opnemen bij de openingstijden zodat bij gebruikers gelijk duidelijk is dat:
> " De gemeente Harderwijk is op eerste en tweede kerstdag gesloten"
Deze opmerking moet onder de openingstijden van een lokatie kunnen worden ingevoerd en getoond onder de openingstijden van een lokatie.

### PR DESCRIPTION
**Pull request by dariomory**
Fixes #236

_______

**Changes**

- Allow commenting on opening hours.
- Added opening times to location serializer.

### CODE REVIEW

File: src/sdg/organisaties/models.py [hunk 1]
```diff
@@ -110,19 +110,22 @@ class Lokatie(models.Model):
...
+        help_text=_("De straatnaam van de lokatie."),
     )
     nummer = models.PositiveIntegerField(
         _("nummer"),
+        help_text=_("Het huisnummer van de lokatie."),
+    )
+    plaats = models.CharField(
+        _("plaats"),
+        max_length=256,
+        help_text=_("De plaatsnaam van de lokatie."),
```
Comment (joeribekker) [inline src/sdg/organisaties/models.py:122]: Mot sure why you moved plaats. The order was good here.

Dutch addresses are typically:

```
<straat> <nummer>
<postcode> <plaats>
```

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: src/sdg/organisaties/models.py (hunk 1 above)
Comment (dariomory) [reply src/sdg/organisaties/models.py:122]: The reason for this change is due to the ordering on the form:

location form

File: src/sdg/templates/organisaties/update.html [hunk 2]
```diff
@@ -76,8 +77,4 @@ <h3 class="form__subtitle">
     <template class="formset__template">
         {% include "organisaties/_prefix/lokatie_formset.html" %}
     </template>
-
-    <div class="form__actions">
-        <p class="form__action-help">Wijzigingen worden direct gepubliceerd</p>
-    </div>
```
Comment (joeribekker) [inline src/sdg/templates/organisaties/update.html:82]: Why was this removed?

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: src/sdg/templates/organisaties/update.html (hunk 2 above)
Comment (dariomory) [reply src/sdg/templates/organisaties/update.html:82]: Reason of removal is because it was part of the initial mock and it seemed to be superfluous and inconsistent after the re-design updates. It can be re-added with some minor adjustments.
